.vscode/cspell.json

```diff
@@ -25,7 +25,8 @@
     "*nlohmann-json*",
     "eng/docs/api/assets/**/*",
     "eng/CredScanSuppression.json",
-    "**/test/ut/recordings/*.json"
+    "**/test/ut/recordings/*.json",
+    "**/test/ut/test_base.hpp"
```

Comment:
This file is part of the source.  It needs to be scanned as well.
```suggestion
```

Reply:
Jinx (same comment as the following):
https://github.com/Azure/azure-sdk-for-cpp/pull/4118#discussion_r1044912974

Reply:
done